### PR TITLE
3 get/api/topics

### PR DESCRIPTION
**Pull request by smpezert**
3-get completed

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  test("status:200, responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              description: expect.any(String),
+              slug: expect.any(String),
```
Comment (P-Copley) [inline __tests__/app.test.js:38]: Lovely test 👋 

Consider adding a length check to make sure that you're not getting back an empty array as your forEach will not run if it's empty.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use("*", (req, res) => {
+  res.status(404).send({ msg: "Invalid path" });
+});
+
```
Comment (P-Copley) [inline app.js:13]: Consider adding a 500 internal error handler here as it will be useful in the long run 😁

**Comment by P-Copley** [comment]
Looking great @smpezert 

Take a look at the above comments and merge when you're ready 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (smpezert) [reply __tests__/app.test.js:38]: i added length property test and typeof test! Thank you!

File: app.js (hunk 2 above)
Comment (smpezert) [reply app.js:13]: I added 500 status. Thank you!

**Pull request merged**